voluncheer/opportunityboard/models.py

```diff
@@ -119,3 +120,7 @@ def duration(self):
         duration = end_datetime - start_datetime
         duration_seconds = dt.timedelta(seconds=duration.seconds)  # remove date
         return duration_seconds
+
+    def delete_recurrences(self):
```

Comment:
Would this delete any siblings that have already been archived?

Reply:
Good point. I'll add a guard here